site/content/docs/5.2/components/navs-tabs.md

```diff
@@ -253,6 +258,11 @@ Take that same HTML, but use `.nav-pills` instead:
 
 Force your `.nav`'s contents to extend the full available width one of two modifier classes. To proportionately fill all available space with your `.nav-item`s, use `.nav-fill`. Notice that all horizontal space is occupied, but not every nav item has the same width.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
```

Comment:
Feel like the concepts behind will be lost.

Reply:
Once again, the problem is not the technical part, but the look of the component used to illustrate this technical part...

Reply:
Review this whole page with the designers.

Reply:
Linked to my previous comment. If this is only a design reason but that the feature can be used it should be clearly explained.

Reply:
OK. From the info I have so far, the problem is the fact that the design should be buttons and not links.
TBH, it would be good to re-clarify the reasons of not using with the design. 
Meanwhile, what about:
> These **link** variants, which are just **examples illustrating the use of the fill and justify utilities**, should not be used because they do not respect the Orange Design System specifications. Indeed, they should be button components.

With more details, it might be useful to add a "refer to our Boosted ..." and/or "refer to the xx guidelines the ODS website"

Reply:
> Here, we have a nav with links that have the look of buttons . Here, it's not possible to have that. If we want to have components that have the look of buttons, then in the code they should be html tags.

I strongly disagree with the fact that `<a>` shouldn't look like buttons. Since many call to action are presented as buttons but are truly links. I think it strongly depends on the context and the associated action. IMO, the actual behavior is fine.
 
Otherwise, many many and many designs will have to change as well.

IMO, if this is accepted by design: 
![image](https://user-images.githubusercontent.com/91960143/224987053-d79a981a-2f8e-4fde-89c6-828d041bc379.png)

instead of:
![image](https://user-images.githubusercontent.com/91960143/224988080-e8252ef0-f7d6-40ea-810a-13a2a3b3db9e.png)

We might consider using `.nav-underline` instead of `.nav-pill` to present the remaining doc. We should keep a callout only if the design is against a specific feature. Any thoughts @julien-deramond ?

Reply:
Just done it.